Header header
DataCube[] cubes
sensor_msgs/Image im